src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

```diff
@@ -311,6 +319,15 @@ To determine which method to use, see [Report mobile monitoring custom events an
 </table>
 
 
+## Query C# Errors(Fatal/Non Fatal).
+
+C# errors and handled exceptions can be seen in the `Handled Exceptions` tab in New Relic One. You will be able to see the event trail, attributes, and stack trace for each C# error recorded.
+
+You can also build a dashboard for these errors using this query:
+
+```sql
+SELECT * FROM MobileHandledException SINCE 24 hours ago
+
```

Comment:
```suggestion
## Query C# Errors [#query-errors]

In the New Relic UI, you can view your fatal and non-fatal C# errors and handled exceptions, including details like event trails, attributes, and stack traces for each recorded error.

<img
  title="Mobile Handled Exceptions"
  alt="Mobile Handled Exceptions"
  src={mobileHandledExceptionsSummary}
/>


To view your errors:
1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**.
2. Select your Unity app.
3. Click **Exceptions > Handled exceptions**. 

You can also build a dashboard for these errors using this query:

```sql
SELECT * FROM MobileHandledException SINCE 24 hours ago
```